PhotDM.tex

```diff
@@ -904,31 +912,29 @@ \subsubsection{PhotometryFilter.transmissionCurve}
 
 \paragraph{PhotometryFilter.transmissionCurve.transmissionPoint} 
 The transmission curve is a mathematical function that describes the transmission 
-fraction of a certain filter in a defined spectral range. This function can be discretised 
```

Comment:
In british english, discretised is correct. I am trying to use British in the text whenever possible

Reply:
The autogeneration failed. Should we correct it first?